infrastructure/variables.tf

```diff
@@ -48,3 +48,12 @@ variable "min_workers" {
     production = 3
   }
 }
+variable "vector_secret_name" {
```

Comment:
Perhaps this should be `postgres_secret_name` to correspond to how we reference it elsewhere

Reply:
I agree. The variable names are a little confusing. Will do a once over and change them .